### PR TITLE
Expose NetServer currentlyKicking for plugin access

### PR DESCRIPTION
**Pull request by BalaM314**
If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [x] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [x] I have ensured that my code compiles, if applicable.
- [x] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

**Comment by BalaM314** [comment]
image
what

**Comment by Phinner** [comment]
Wouldn't it be better to just copy the existing code and implement your own votekick command ?

**Comment by BalaM314** [comment]
I'm trying to copy the "cancel votekick" code, but can't because closure
If you mean copy the entirety of votekick, that is planned but will take a while

**Comment by buthed010203** [comment]
If you're going to make the various kick related variables public, it may be a good idea to make them not final so that servers can change them without writing their own kick logic

File: core/src/mindustry/core/NetServer.java [hunk 1]
```diff
@@ -98,6 +98,15 @@ public class NetServer implements ApplicationListener{
     private boolean closing = false, pvpAutoPaused = true;
     private Interval timer = new Interval(10);
     private IntSet buildHealthChanged = new IntSet();
+    
+    /** Current kick sessions. */
+    public VoteSession[] currentlyKicking = {null};
```
Comment (Anuken) [inline core/src/mindustry/core/NetServer.java:103]: There is not reason to make this an array if it's not a local variable. Also, mark it `@Nullable.`

Comment (BalaM314) [reply core/src/mindustry/core/NetServer.java:103]: Oops, didn't realize it was an array only to allow other code to modify it

**Pull request merged**